documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc

```diff
@@ -0,0 +1,31 @@
+[id='cli_start-{context}']
+= Connecting to {brandname} Servers running in a container
+Establish CLI connections to {brandname} when running in a container.
+
+.Prerequisites
+
+Have at least one running {brandname} server instance in a container.
+
+.Procedure
+
+. Open a terminal
+. Start the CLI.
++
+* *Podman or Docker:*
++
+[source,options="nowrap",subs=attributes+]
+----
+podman/docker run -it --net=host infinispan/cli
+----
+
+. Run the [command]`connect` command and enter your username and password when prompted
```

Comment:
```suggestion
. Run the `connect` command to connect  to a {brandname} server running on the default port of `11222`. Enter your username and password when prompted. 
```
I'm not sure if the "[command]"  is portable to different asciidoctor versions. If you know it is, pleas retain it!